lua/kznllm/init.lua

```diff
@@ -24,12 +24,14 @@ function M.make_prompt_from_template(prompt_template_path, prompt_args)
     command = 'minijinja-cli',
     args = { '-f', 'json', '--lstrip-blocks', '--trim-blocks', prompt_template_path:absolute(), '-' },
     writer = json_data,
-    on_stderr = function(message, _)
-      error(message, 1)
-    end,
   }
 
   active_job:sync()
+  if active_job.code ~= 0 then
+    local error_msg = table.concat(active_job:stderr_result(), '\n')
+    error('[minijinja-cli] (exit code: ' .. active_job.code .. ')\n' .. error_msg, vim.log.levels.ERROR)
```

Comment:
Hi, @chottolabs . I think on current plenary versions its not stderr_buffered but `enable_recording`.

It might be worth double checking because  plenary seems to raise trying to access stderr_result unless the above param is set to true:

https://github.com/nvim-lua/plenary.nvim/blob/ec289423a1693aeae6cd0d503bac2856af74edaa/lua/plenary/job.lua#L464C3-L467C4

Reply:
hmm.. you're right - it's even weirder that it seems to be working like usual on my side tho

Reply:
is this even intended? `enable_recording` =/= `enabled_recording` typo? 🤨

Reply:
yeah so i guess `enable_recording` defaults to true if nil so it never triggered that assert

Reply:
honestly good to make it explicit tho, I will add it in for consistency